helmfile/01-applications.yaml

```diff
@@ -435,7 +435,7 @@ releases:
 
 {{ if .Values.user.alertmanager.enabled }}
 - name: user-alertmanager
-  namespace: monitoring
+  namespace: {{ .Values.user.alertmanager.namespace | default "monitoring" }}
```

Comment:
The default value is to eliminate error when a user sets `user.alertmanager.namespace` to null.

Reply:
Have you tried setting it to null? I've had to use something like `get .Values | user.alertmanager...` to allow values to be missing.
Maybe it's different if the value exists but is unset/null though :slightly_smiling_face:

Reply:
Yes, if you set n`amespace` to null (`namespace:`) or empty `namespace: ""` the apply will fail